react/components/RequestOrganizationForm.tsx

```diff
@@ -208,25 +208,6 @@ const RequestOrganizationForm: FC = () => {
     setCostCenterCustomFieldsState(costCenterFieldsToDisplay)
   }, [defaultCustomFieldsData])
 
-  const handleOrgCustomFieldsUpdate = (
```

Comment:
The code removed here was duplicated for the org creation within the admin and Sonar was complaining about it, failing the build. I moved it to the new `CustomFieldInputList` component and now Sonar is passing (and we have less duplicated code).

I've tested this part of the UI a lot, both in the admin interface and in the storefront one (which is the org creation request page) and both are working exactly as before, for both org and cost center custom fields.